src/checks/record/pass-schema.ts

```diff
@@ -13,7 +13,7 @@ const check: Check = {
     url: 'https://github.com/datenanfragen/data#data-formats',
     severity: 'ERROR',
     run: (json, ctx) => {
-        const schema = ctx.file_path.startsWith('supervisory-authorities/') ? adb_schema : cdb_schema;
+        const schema = /^supervisory-authorities[\\/].*/.test(ctx.file_path) ? adb_schema : cdb_schema;
```

Comment:
Haven't tested it, but shouldn't that also suffice?

```suggestion
        const schema = /^supervisory-authorities[\\/]/.test(ctx.file_path) ? adb_schema : cdb_schema;
```